server/src/models/user.js

```diff
@@ -0,0 +1,47 @@
+import Joi from 'joi';
+import mongoose from 'mongoose';
+
+const userSchema = new mongoose.Schema({
+  given_name: {
+    type: String,
+    minlength: 2,
+    maxlength: 50,
+  },
+  family_name: {
+    type: String,
+    minlength: 2,
+    maxlength: 50,
+  },
+  email: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 50,
+    unique: true,
+  },
+  name: {
+    type: String,
+    minlength: 2,
+    maxlength: 100,
+  },
+  picture: {
+    type: String,
+  },
+});
+
+const User = mongoose.model('User', userSchema);
+
+function validateUser(user) {
+  const joiSchema = {
+    given_name: Joi.string().min(2).max(50).required(),
```

Comment:
Which typecase are you guys following?

Reply:
camelcasing for variable names,
PascalCase for component/class naming

Reply:
will update the property names with an underscore to camelcase.